src/ops.jl

```diff
@@ -46,8 +46,11 @@ function onnx_flatten(x; axis = 1)
 end
 
 add(xs...) = .+(xs...)
+sub(xs...) = .-(xs...)
 mul(xs...) = .*(xs...)
 relu(x) = NNlib.relu.(x)
+elu(x) = NNlib.elu.(x)
+tanh(x) = NNlib.tanh.(x)
```

Comment:
```suggestion
tanh(x) = tanh.(x)
```
Since `NNlib.tanh === Base.tanh`.